jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,176 @@
+`GET /workflow/:id ` | This endpoint enables users to return an individual Workflow based on the `id` parameter being passed in the request
+`GET /workflow/:id/jobs` | This endoint enables users to retrieve all Jobs associated with a specific workflow, based on its unique `id`.
+`GET /project/:project_slug`  | This endpoint enables users to retrieve a specific Project by its unique slug.
+`POST /project/:project_slug/pipeline` | This endpoint enables users to retrieve an individual project by its unique slug.
+`GET /pipeline/:id` | This endpoint enables users to retrieve an individual pipeline, based on the `id` passed in the request.
+`GET /pipeline/:id/config`  | This endpoint enables users to retrieve the configuration of a specific pipeline.
+`GET /project/:project_slug/pipelines/[:filter]`  | This endpoint enables users to retrieve the most recent set of pipelines for a Project.
+
+## API Syntax
+
+When making an API request, make sure you follow standard REST API syntax and formatting. Adhering to proper REST API syntax ensures that the API server can properly process your request and return a valid JSON response. To make a request to the CircleCI API, use the following format:
+
+`https://circleci.com/api/v2`
+
+Where:
+
+- `https://circleci.com` is the resource URL for the API being called.
+- `api` is the class being called.
+- `v2` is the API version.
+
+# Getting Started with the API
+
+The CircleCI API v2 is backwards-compatible with previous API versions in the way it identifies your projects using repository name. For instance, if you want to pull information from CircleCI about the GitHub repository https://github.com/CircleCI-Public/circleci-cli you can refer to that in the CircleCI API as gh/CircleCI-Public/circleci-cli, which is a “triplet” of the project type, the name of your “organization”, and the name of the repository. For the project type you can use `  github` or `bitbucket` as well as the shorter forms `gh` or `bb`, which is supported in API v2. The `organization` is your username or organization name in your version control system.
+
+With API v2, CircleCI is introducing a string representation of the triplet called the `project_slug`, which takes the following form:
+
+`<project_type>/<org_name>/<repo_name>`
+
+The `project_slug` is included in the payload when pulling information about a project as well as when looking up a pipeline or workflow by ID. The `project_slug` can then be used to get information about the project. It is possible in the future the shape of a project_slug may change, but in all cases it would be usable as a human-readable identifier for a given project.
+
+## Rate Limiting/Throttling
+
+CircleCI does not currently perform any throttling or rate limiting. In the future, CircleCI may institute rate limiting or throttling or both to monitor the number of API calls made to the service and ensure that the system is more stable and has higher overall performance
```

Comment:
I don't think we should say this. Invites abuse.

Reply:
Removed from document